lib/metadata.js

```diff
@@ -71,19 +70,15 @@ Metadata.prototype.updateOidcMetadata = function updateOidcMetadata(doc, next) {
 
   const self = this;
 
-  self.oidc = objectTransform({
-    source: doc,
-    map: {
-      id_token_signing_alg_values_supported: 'algorithms',
-      authorization_endpoint: 'auth_endpoint',
-    },
-    pick: [
-      'end_session_endpoint',
-      'issuer',
-      'token_endpoint',
-      'userinfo_endpoint',
-    ],
-  });
+  var oidc = {};
+  oidc.algorithms = doc.id_token_signing_alg_values_supported;
```

Comment:
any particular reason for a name difference between 'doc' and 'oidc'? For example why auth_endpoint and authorization_endpoint (which is the name in the spec). I would stick with the names in the spec, otherwise we are always translating. Sometimes, we may have overlap.  Check out the names in: https://login.microsoftonline.com/common/.well-known/openid-configuration